src/utils/tests/generateEventLink.test.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    CalendarEvent,
+    CalendarType,
+    generateEventLink,
+} from '../generateEventLink';
+
+describe('generateEventLink', () => {
+    // UTC is used to be indifferent to locales
+    const startDate = new Date(Date.UTC(2020, 11, 20, 16, 30, 0));
```

Comment:
thank you for the fix, but I think we should not use here Date instances, we should work only with CalendarEvent object, bcz this is what will come after the quiz. I agree with you about [this comment](https://github.com/event-maker-telegram-bot/event-maker-telegram-bot/pull/25/files#r506040314), the bug is there

Reply:
Yes, I agree we need to replicate real user behaviour instead of trying to satisfy CI's environment. Actually the snapshot test revealed the real problem. I introduced the fix more as part of my research, you may revert it.

Reply:
but thanks for idea `Date.UTC`, it solves the issue